README.md

```diff
@@ -1,308 +1,60 @@
-# kubemacpool
+# Kubemacpool
 
-## About
+Kubemacpool provides collision-aware MAC Address allocation for Pods and VirtualMachines in a Kubernetes cluster operating 
+with multiple CNI plugins, using [Network Plumbing Working Group de-facto standard](https://github.com/k8snetworkplumbingwg/multi-net-spec).
 
-This project allow to allocate mac addresses from a pool to secondary interfaces using
-[Network Plumbing Working Group de-facto standard](https://github.com/k8snetworkplumbingwg/multi-net-spec).
-For VirtualMachines, it also allocates a mac address for the primary interface if [masquerade](https://kubevirt.io/user-guide/#/creation/interfaces-and-networks?id=masquerade) or [slirp](https://kubevirt.io/user-guide/#/creation/interfaces-and-networks?id=slirp) binding mechanism is used.
+## Deployment
 
-## Usage
+See the deployment guide to learn how to install Kubemacpool on your [arbitrary cluster](doc/deployment-on-arbitrary-cluster.md).
 
-For test environment you can use the [development environment](#Develop).
+In case you just want to try it out, consider spinning up a [local virtualized cluster](doc/deployment-on-local-cluster.md).
 
-For Production deployment:
+**Note** For VirtualMachines, Kubemacpool supports primary interface MAC Address allocation for [masquerade](https://kubevirt.io/user-guide/#/creation/interfaces-and-networks?id=masquerade) or [slirp](https://kubevirt.io/user-guide/#/creation/interfaces-and-networks?id=slirp) binding mechanisms.
```

Comment:
Let's align on a single `**Note**` lettering and usage of colon. There are couple of other places with notes that use different typography. Personally, I usually tend to use `**Note:**`.

Reply:
Done